src/plugins/console/server/routes/api/console/proxy/tests/query_string.test.ts

```diff
@@ -62,25 +63,22 @@ describe('Console Proxy Route', () => {
       describe('contains full url', () => {
         it('treats the url as a path', async () => {
           await request('GET', 'http://evil.com/test');
-          expect((requestModule.proxyRequest as jest.Mock).mock.calls.length).to.be(1);
-          const [[args]] = (requestModule.proxyRequest as jest.Mock).mock.calls;
-          expect(args.uri.href).to.be('http://localhost:9200/http://evil.com/test?pretty=true');
+          const [[args]] = opensearchClient.asCurrentUser.transport.request.mock.calls;
+          expect(args.path).toBe('/http://evil.com/test?pretty=true');
         });
       });
       describe('starts with a slash', () => {
-        it('combines well with the base url', async () => {
+        it('keeps as it is', async () => {
           await request('GET', '/index/id');
-          expect((requestModule.proxyRequest as jest.Mock).mock.calls.length).to.be(1);
-          const [[args]] = (requestModule.proxyRequest as jest.Mock).mock.calls;
-          expect(args.uri.href).to.be('http://localhost:9200/index/id?pretty=true');
+          const [[args]] = opensearchClient.asCurrentUser.transport.request.mock.calls;
+          expect(args.path).toBe('/index/id?pretty=true');
         });
       });
       describe(`doesn't start with a slash`, () => {
-        it('combines well with the base url', async () => {
+        it('adds slash to path before sending request', async () => {
           await request('GET', 'index/id');
-          expect((requestModule.proxyRequest as jest.Mock).mock.calls.length).to.be(1);
-          const [[args]] = (requestModule.proxyRequest as jest.Mock).mock.calls;
-          expect(args.uri.href).to.be('http://localhost:9200/index/id?pretty=true');
+          const [[args]] = opensearchClient.asCurrentUser.transport.request.mock.calls;
+          expect(args.path).toBe('/index/id?pretty=true');
```

Comment:
Just a question, why does it not append to the base url anymore?

Reply:
synced offline, base url are determined at the client init phase, we no longer need to pass it to dev tool console logic explicitly